### ISSUE TITLE
Inference on pretrained model

### ISSUE THREAD

**Issue by enrico310786** [whwu95/Cap4Video#27]
Hi, 

your work is very nice. 

I wonder if it would be possible for you to share the pre-trained model and instructions on how to use it. At least the instructions on how to use the pre-trained model on a test set of data. For example given a set of 20 videos, their captions and a query text, retrive the first video closer to the query text.

Thank you!
